src/types/network-monitor.ts

```diff
@@ -91,10 +94,30 @@ export type TransactionParams = {
   interval?: number
 }
 
-export interface InterestingTransaction {
+// NOTICE: blockProcessingVersion V1
+export interface InterestingLog {
   bloomId: string
   transaction: TransactionResponse
   receipt?: TransactionReceipt
   log?: Log
   allLogs?: Log[]
 }
+
+// NOTICE: blockProcessingVersion V2
+export interface InterestingTransaction {
+  transaction: TransactionResponse
+  receipt?: TransactionReceipt
+  allLogs: Log[]
+}
+export interface SqsEvent {
+  sqsEventName: SqsEventName
+  decodedEvent: DecodedEvent | null
+}
+export interface InterestingEvent {
```

Comment:
kind of confused. is `interestingLog` the same thing as an `InterestingEvent`?

Reply:
No, but the goal is for the `InterestingEvent` (V2) to replace the `InterestingLog` (V1).

Reply:
ahh interesting